### PR TITLE
another dictonary expansion

### PR DESCRIPTION
**Pull request by ldue**
Enable ronny to call his friends for help!

### CODE REVIEW

File: src/ronny.js [hunk 1]
```diff
@@ -34,8 +34,13 @@ bot.on('message', function(data) {
           dict.forEach(d => {
             if (d.keys.some(k => ~msg.indexOf(k))) {
               var message = d.message;
+              var params = default_params;
               if (typeof message === 'function')
                 message = message();
+              if (message.params) {
+                params = message.params;
```
Comment (schrej) [inline src/ronny.js]: These should be merged with the default params so we don't have to specify all of them every time. Maybe there are helpers in the lodash package which consists of a ton of utility functions.

Comment (ldue) [inline src/ronny.js]: Done.

Comment (schrej) [inline src/ronny.js]: This won't work with nested values. Will this suffice for our usecase?

File: src/ronny.js [hunk 2]
```diff
@@ -38,7 +38,7 @@ bot.on('message', function(data) {
               if (typeof message === 'function')
                 message = message();
               if (message.params) {
-                params = message.params;
+                for (var attrname in message.params) { params[attrname] = message.params[attrname];}
```
Comment (schrej) [inline src/ronny.js]: this will not work for nested values, correct?
Will this suffice for our usecase?

Comment (ldue) [inline src/ronny.js]: I think that should work for most cases. 
Attachments are the only nested values for the slack api parameters.

Comment (schrej) [inline src/ronny.js]: ok

**Pull request merged**